### DISCUSSION TITLE
Loading a custom mame.ini

### DISCUSSION THREAD

**Discussion by XargonWan** [mamedev/discussions#93] [Q&A] [answered]
Greetings,

thanks for the previous given support now I can correctl compile mame in RetroDECK.
However I got another issue:

**how can I load a custom config file?**
My config files are in `/var/config/mame`, but if I launch just `mame` it cannot find it, so I tried the following:

```
ls /var/config/mame
default.cfg  ___empty.cfg  hiscore  mame.ini  ui.ini
mame -cfg_directory /var/config/mame
```

But is seems like that that mame.ini is ignored as if I check from the mame UI the paths are not the ones I set in the config file, for example my roms are not seen.
Is there any way to force mame to load the config from a custom directory?
We cannot really write in `~/.mame`

Thanks in advance

**Comment by cuavas** [comment, marked as answer]
Because the [-inipath](https://docs.mamedev.org/commandline/commandline-all.html#mame-commandline-inipath) and [-cfg_directory](https://docs.mamedev.org/commandline/commandline-all.html#mame-commandline-cfgdirectory) options do different things.  Please read the documentation.

**Reply by XargonWan** [reply]
This did the trick, thanks, I completely skipped that option.
